### PR TITLE
Meets Bug #2280: Spent Time not localized

### PR DESCRIPTION
**Pull request by netfighter**
```
* `#2280` Fix: Spent Time not localized
```

### CODE REVIEW

**Comment by netfighter** [comment]
Ticket: https://www.openproject.org/work_packages/2280

File: features/work_packages/error_on_update.feature [hunk 1]
```diff
@@ -58,3 +58,27 @@ Feature: Error messages are displayed
      And I submit the form by the "Submit" button
 
     Then I should see an error explanation stating "Subject can't be blank"
+
+  @javascript
+  Scenario: Adding a localized time entry
```
Comment (hschink) [inline features/work_packages/error_on_update.feature]: @netfighter You were right when you asked me whether this cuke belongs here: It does not, sorry. You may move this cuke to `features/work_packages/update.feature` or a even more appropriated place. Thanks! :smile:

**Comment by hschink** [comment]
@netfighter there are two cukes that fail:
1. `features/work_packages/work_package_show.feature:86`
2. `features/work_packages/work_package_textile_link.feature:62`

**Comment by hschink** [comment]
@netfighter can you please rebase the branch? There are some commits that do not belong here. Thank you! :smiley:

**Comment by netfighter** [comment]
done

**Comment by hschink** [comment]
Added PR manually:
1. [034eb0c](https://github.com/opf/openproject/commit/034eb0c8f6833fa730daa2c08fa779d9df9a47aa)
2. [a195557](https://github.com/opf/openproject/commit/a1955579f83ed1c8f9f925e9ff7e704b248d3c92)

**Pull request closed without merging**